pyvista/core/dataset.py

```diff
@@ -2083,7 +2083,10 @@ def find_closest_point(self, point: Iterable[float], n=1) -> int:
             return vtk_id_list_to_array(id_list)
         return locator.FindClosestPoint(point)
 
-    def find_closest_cell(self, point: Union[int, np.ndarray]) -> Union[int, np.ndarray]:
+    def find_closest_cell(self,
+                          point: Union[int, np.ndarray],
```

Comment:
As pointed out in https://github.com/pyvista/pyvista/pull/2036#discussion_r787057591, this type is incorrect for `point`.